src/identity/core/getName.ts

```diff
@@ -14,5 +13,5 @@ export const getName = async ({
     address,
   });
 
-  return ensName ?? getSlicedAddress(address);
+  return ensName ?? null;
```

Comment:
This introduce a breaking change that we should document in the Changeset.

Reply:
updated the changeset